diagnostic.md

```diff
@@ -58,13 +62,17 @@ Suppose you want to define a mixin named `row` stored in `./row.scss`. Write the
 code to import the mixin definition in the current module.
 
 ```scss
-// your response here
+@import './row.scss';
 ```
 
 Now that the mixin is imported, let's use it. This mixin doesn't take any
 arguments. Write the code to include the mixin inside all elements with a
 class of `content`.
 
 ```scss
-// your response here
+@mixin row(){
+  @include .content().val();
+}
+
+I'm not sure I understood the question
 ```
```

Comment:
Assume in `row.scss`, we defined a mixin called `row`. Similar to what you created for your answer. For this answer, though, we just needed you to call that mixin, like so:

```CSS
.content {
  @include row;
}
```